templates/tab/js/non-sso/src/index.css

```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+  /* outline: 1px dashed #f00; */
```

Comment:
If it is not used, you can remove it

Reply:
Here I prefer to keep the same as existing template.

Reply:
Sure